.travis.yml

```diff
@@ -45,10 +45,10 @@ script:
  - ${TRAVIS_BUILD_DIR}/.travis/check-formatting.sh
  - go get -d -t -v ./...
  - go build -v ./...
- - go test -v ./...
```

Comment:
Please revert this. This is useful for me when we are debugging failures.

Reply:
what part do you want reverted? dropping the `-a` ? (but then this defeats the Go builtin cache mechanism, which is now (?) robust)

Reply:
The `-v`. Note that it's difficult to review when the file state go away with force push.

Reply:
running 4 `go test` with `-v` generates way too many lines for Travis (it ends up cutting them in the web display, allowing one to download the raw log file.)

in general, I find that while `-v` for `go (get|install|build)` is fine, for `go test` it usually is wrong after ~30 tests.

Reply:
Which is one of the reason why I want the matrix with TAGS back in https://github.com/gonum/gonum/pull/800#discussion_r248054735.